README.markdown

```diff
@@ -193,6 +193,15 @@ class Application < Rails::Application
 end
 ```
 
+### I'm getting "undefined method 'web_console'"
+
+Configuration should be set in a dedicated environment, not in the
+application.rb.
+Move your web-console configuration to the files under config/environment.
```

Comment:
Can you mention `config/environments/development.rb` explicitly. This is the configuration file we want to advertise our configuration for.